modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -379,6 +385,19 @@ public String toApiDocFilename(String name) {
         return toApiName(name);
     }
 
+    @Override
+    public String addRegularExpressionDelimiter(String pattern) {
+        if (StringUtils.isEmpty(pattern)) {
+            return pattern;
+        }
+
+        if (!pattern.matches("^/.*")) {
```

Comment:
Hmm, I tried removing the `.*` and re-running my unit test, but I'm getting a couple of failures. I think it does need the `.*`, otherwise it will check to see if the pattern literally matches the string `"/"` and nothing else. Having the `.*` there says match any string that starts with `/` and has any number of characters afterwards.

Reply:
Sorry, you are right.